CIP-1694/README.md

```diff
@@ -0,0 +1,1163 @@
+> (including to a pre-defined DRep), the rewards can be withdrawn.
+
+#### DRep incentives
+
+DReps arguably need to be compensated for their work. Research on incentive models is still ongoing,
+and we do not wish to hold up implementation of this CIP while this is resolved.
+
+Our interim proposal is therefore to escrow Lovelace from the existing Cardano treasury until this
+extremely important decision can be agreed on by the community, through the on-chain governance
+mechanism that is being constructed.
+
+Alternatively, DReps could pay themselves through instances of the "Treasury withdrawal" governance action.
+Such an action would be auditable on-chain, and should reflect an off-chain agreement between DReps and delegators.
+
+<!---------------------------           DReps          ------------------------>
+<!---------------------------    Governance Actions    ------------------------>
+
+### Governance actions
+
+We define seven different types of **governance actions**.
+A governance action is an on-chain event that is triggered by a transaction and has a deadline after which it cannot be enacted.
+
+- An action is said to be **ratified** when it gathers enough votes in its favor (through the rules and parameters that are detailed below).
+- An action that doesn't collect sufficient `Yes` votes before its deadline is said to have **expired**.
+- An action that has been ratified is said to be **enacted** once it has been activated on the network.
+
+Regardless of whether they have been ratified, actions may, however, be **dropped without being enacted** if,
+for example, a motion of no confidence is enacted.
+
+
+| Action                                             | Description |
+| :---                                               | :--- |
+| 1. Motion of no-confidence                         | A motion to create a _state of no-confidence_ in the current constitutional committee |
+| 2. New constitutional committee and/or quorum size | Changes to the members of the constitutional committee and/or to its signature threshold |
+| 3. Updates to the Constitution                     | A modification to the off-chain Constitution, recorded as an on-chain hash of the text document |
+| 4. Hard-Fork[^2] Initiation                        | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
+| 5. Protocol Parameter Changes                      | Any change to one or more updatable protocol parameters, excluding changes to major protocol versions ("hard forks") |
+| 6. Treasury Withdrawals                            | Movements from the treasury, sub-categorized into small, medium or large withdrawals (based on the amount of Lovelace to be withdrawn). The thresholds for treasury withdrawals are discussed below. |
+| 7. Info                                            | An action that has no effect on-chain, other than an on-chain record. |
```

Comment:
What would this be used for? A community notes of sorts to be recoded on chain for eternity? Seems a bit pointless.

Reply:
https://github.com/JaredCorduan/CIPs/blob/voltaire-v1/CIP-1694/README.md#the-versatility-of-the-info-governance-action